pkg/qtv/downstream_client_commands.go

```diff
@@ -245,7 +245,7 @@ func (ds *dStream) soundListClientCmd(tr *tokenizerResult) (err error) {
 		return ds.setState(dsNeedInitialData)
 	}
 
-	if err := us.qp.sendList(ds, us.qp.soundList[:], svc_soundlist); err != nil {
+	if err := us.qp.sendList(ds, us.qp.soundList[:], svc_soundlist, svc_fte_soundlistshort_UNUSED); err != nil {
```

Comment:
For symmetry, but in practice will never surpass svc_soundlist as svc_fte_soundlistshort is not enabled.